azure/scripts/playbooks/roles/ocpcluster/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+---         
+- name: "Download OpenShift Binaries"
+  ansible.builtin.shell: |
+    runuser -l "{{ BOOTSTRAP_ADMIN_USERNAME }}" -c "wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-{{ OPENSHIFT_VERSION }}/openshift-install-linux.tar.gz"
+    runuser -l "{{ BOOTSTRAP_ADMIN_USERNAME }}" -c "wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-{{ OPENSHIFT_VERSION }}/openshift-client-linux.tar.gz"
+    chown "{{ BOOTSTRAP_ADMIN_USERNAME }}":"{{ BOOTSTRAP_ADMIN_USERNAME }}" "{{ INSTALLER_HOME }}"
```

Comment:
For example, the ansible [file module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html) should be used instead of `chown` or `chmod`.